ivy_tests/test_ivy/test_frontends/test_tensorflow/test_activations.py

```diff
@@ -106,3 +106,85 @@ def test_tensorflow_softmax(
         x=x[0],
         axis=axis,
     )
+
+
+# elu
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=-3,
+        max_value=3,
+        min_num_dims=1,
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=3,
+    ),
+    alpha=st.one_of(
+        helpers.floats(
+            min_value=-3,
+            max_value=3,
+        )
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.elu"

```

Comment:
The file structure in the TF frontends has been changed in a recent commit which is causing the tests to fail. The correct namespace would now be 
`ivy.functional.frontends.tensorflow.keras.elu`
Could you kindly make this small change 😊